src/components/Nutrition.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import cx from "classnames";
+
+//Displays additional nutritional values
+class Nutrition extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      showNutrition: false
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+  //Displays nutritional info
+  handleClick(event) {
+    event.preventDefault();
+
+    this.setState({
+      showNutrition: !this.state.showNutrition
+    });
+  }
+
+  render() {
+    const buttonClassSwitch = cx("nutrition", {
+      "nutrition--on": this.state.showNutrition == true
```

Comment:
`"nutrition--on": this.state.showNutrition` would work better here since showNutrition gets evaluated as a boolean